server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -366,11 +366,13 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 
 		const enableWholeSummaryUpload = config.get("storage:enableWholeSummaryUpload") as boolean;
 		const opsCollection = await databaseManager.getDeltaCollection(undefined, undefined);
+		const storageNameProvider = customizations?.documentRepository ?? new services.StorageNameProvider(tenantManager);
```

Comment:
How is storageNameProvider related with documentRepository ?

Reply:
Copy past error 😢

Reply:
Removed this change by current design